src/main/java/com/insidious/plugin/client/SessionInstance.java

```diff
@@ -2409,7 +2409,17 @@ private void scanDataAndBuildReplay() {
             properties.put("session", executionSession.getPath());
             properties.put("message", e.getMessage());
             try {
-                properties.put("stacktrace", ObjectMapperInstance.getInstance().writeValueAsBytes(e.getStackTrace()));
+                StackTraceElement[] stackTrace = e.getStackTrace();
+                StringBuilder stackTraceBuilder = new StringBuilder();
+                for (int i=0;i<=stackTrace.length-1;i++) {
+                    stackTraceBuilder.append("__")
+                            .append(stackTrace[i].getFileName())
+                            .append("_")
+                            .append(stackTrace[i].getLineNumber());
+                }
+                stackTraceBuilder.append("ENDS");
+
+                properties.put("stacktrace", ObjectMapperInstance.getInstance().writeValueAsString(stackTraceBuilder.toString()));
```

Comment:
stackTraceBuilder is just a string so why does it need object mapper to write value as string

Reply:
Yes, that is not needed anymore. Removed that as well as the try-catch block for JsonProcessingException from ObjectMapper